Author: Daniel Baumann <daniel@debian.org>
Description: Replaces xterm with x-terminal-emulator.

diff -Naurp dwm.orig/config.def.h dwm/config.def.h
--- dwm.orig/config.def.h	2008-06-19 10:11:38.000000000 +0200
+++ dwm/config.def.h	2008-06-23 08:54:16.000000000 +0200
@@ -49,7 +49,7 @@ static Layout layouts[] = {
 
 /* commands */
 static const char *dmenucmd[] = { "dmenu_run", "-fn", font, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
-static const char *termcmd[]  = { "uxterm", NULL };
+static const char *termcmd[]  = { "x-terminal-emulator", NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
